changefeedtrigger/__init__.py

```diff
@@ -0,0 +1,94 @@
+import time
+import json
+from subprocess import Popen, PIPE
+from azure.storage.blob.changefeed import ChangeFeedClient
+from azure.storage.blob import BlobServiceClient
+import azure.functions as func
+from datetime import timedelta, datetime
+import logging
+from predicate import where
+
+
+def multidimensional_filtering(cf_client, b_filter, blob_client, cont_name, c_filter, e_filter):
+    filter_event_service = blob_client.get_container_client(container=cont_name)
+    blob_name = "continuation-token"
+
+    # access blobs in container
+    blob_list = filter_event_service.list_blobs()
+    tokens = []
+    for blob in blob_list:
+        tokens.append(blob.name)
+    
+    # check if continuation token already exists
+    if blob_name in tokens:
+        cursor = filter_event_service.download_blob(blob_name).readall()
+        # if value of token is None return None
+        if cursor == b"None":
+            return None
+        # if value of token is a cursor initiate the changefeed with cursor
+        elif cursor != b"" and cursor != b"None":
```

Comment:
I think we can safely assume that if blob exists then content is cursor.

Reply:
Basically i was running into an issue that when the cursor gets to the very end of all of the events, it does not store as the last event it stores as None, so I was doing this to navigate that situation! however, i can probably get rid of the check for if it is just an empty string

Reply:
that sounds like a bug in CF impl then. if CF gets to the end it should still produce a cursor that can be used to resume later if new events were produced. @xiafu-msft

Reply:
right now i have changed it so that the changefeed only iterates one event per page. that won't fix the issue but will hopefully make it more manageable

Reply:
I'd rather not go for one event page. Instead, I'd just dump last non-empty cursor and allow some events to be processed twice as a worst case. Until the CF impl is fixed.

Reply:
I just updated it so that if the cursor will be None it just repeats the last run over and over with the previous cursor until an entire page of events can be fulfilled. This way it can cover more than 1 event per page-- i added choosing how many per page to the top so it can be customized. Also, with this change it should be able to actually run continuously and reupdate once new events are added.